lib/async_hooks.js

```diff
@@ -359,9 +354,9 @@ function emitHookFactory(symbol, name) {
     } catch (e) {
       fatalError(e);
     }
-    processing_hook = false;
+    processing_hook -= 1;
```

Comment:
nit: Since we are going with depth counting, IMHO the decrement should be in a `finally` clause